addons/account/models/chart_template.py

```diff
@@ -168,6 +168,13 @@ def generate_journals(self, acc_template_ref, company, journals_dict=None):
         JournalObj = self.env['account.journal']
         for vals_journal in self._prepare_all_journals(acc_template_ref, company, journals_dict=journals_dict):
             journal = JournalObj.create(vals_journal)
+            if journal.type == 'purchase':
+                # Check if any journal already configured with vendor bill alias or not
+                journal_with_alias = JournalObj.search([
+                    ('type', '=', 'purchase'), ('alias_prefix', '!=', False), ('company_id', '=', journal.company_id.id)
+                ], limit=1)
+                if not journal_with_alias:
+                    journal.alias_id.alias_name = 'vendorbills'
```

Comment:
will never happens, so let's hardcode that default value

Reply:
Never set default value to alias in an hidden way. It depends heavily on how you configured your emial server and will likely lead to lost emails.

Reply:
so? how do you set the default value, then?

Reply:
Better should be to have a unique alias that you can define in the application settings. Normally this is the standard way of doing it. It was designed like that in expense or lead I think.

Reply:
@tde-banana-odoo Initial spec was just like that("Better should be to have a unique alias or a default alias name / prefix that you can define in the application settings...") but updated later and hence we changed the dev according to it.